docs/modules/plugins/pages/plugin-db.adoc

```diff
@@ -354,3 +354,135 @@ specified database using SQL query.
 ----
 {transformer=FROM_DB, dbKey=testh2db, sqlQuery=SELECT * FROM students, nullReplacement= }
 ----
+
+== Steps
+
+=== Execute SQL query and save its result
+
+Executes SQL query against specified database and saves the query result to a variable.
+
+[source,gherkin]
+----
+When I execute SQL query `$sqlQuery` against `$dbKey` and save result to $scopes variable `$variableName`
+----
+
+* `$sqlQuery` - https://www.techtarget.com/searchdatamanagement/definition/SQL[The SQL query to execute].
+* `$dbKey` - <<_database_connection, The database connection key>>.
+* `$scopes` - xref:commons:variables.adoc#_scopes[The comma-separated set of the variables scopes].
+* `$variableName` - The variable name to store the SQL query result.
+
+.The launch_rockets table available by 'launches' db connection
+[source]
+----
+|id|name        |payload_cost|country|
+|1 |Falcon 9    |$2,720      |USA    |
+|2 |Zhongxing-19|$4,320      |CHINA  |
+----
+
+.Query USA rocket launches and validate result set
+[source,gherkin]
+----
+When I execute SQL query `SELECT * FROM launch_rockets WHERE country='USA'` against `launches` and save result to scenario variable `usa-launches`
+Then `usa-launches` is equal to `[{
+    name=Falcon 9,
+    payload_cost=$2,720,
+    country=USA
+}]`
+----
+
+The individual rows are available by index starting from zero as per the following example:
+
+.Query USA rocket launches and check name in first row
+[source,gherkin]
+----
+When I execute SQL query `SELECT * FROM launch_rockets WHERE country='USA'` against `launches` and save result to scenario variable `usa-launches`
```

Comment:
Please format SQL query

Reply:
Formatted for cases where we have where statement, just SELECT * FROM launch_rockets were not formatted